sycl/test-e2e/ESIMD/api/svm_gather_scatter.cpp

```diff
@@ -54,7 +54,7 @@ template <typename T, int N> bool test(queue &Q) {
   T *Dst = Ptr2.get();
 
   for (int I = 0; I < N; ++I) {
-    Src[I] = I + 1;
+    Src[I] = I + 1 + 0.5;
```

Comment:
Sorry can you explain this change?

Reply:
I did not like the fact the data is basically int. This change is the attempt to make the test a bit more strict and catch the cases where our API implementation can use fp-to-int vector conversion where fp-to-int and back bit-cast was needed.